spec/defaultHtmlSequencerUi.spec.js

```diff
@@ -1,7 +1,7 @@
 describe('Default sequencer HTML', function() {
 
   var DefaultHtmlSequencerUi = require('../examples/lib/defaultHtmlSequencerUi');
-  var sequencer = require('../src/ImageSequencer')();
+  var sequencer = require('../src/ImageSequencer')({useWasm:true});
```

Comment:
Is this still necessary since it's default on now?

Reply:
Ya I corrected almost all the files, just left here!!
Correcting now!!